test/integration_test.coffee

```diff
@@ -18,7 +18,7 @@ describe 'Integration tests for the 3Scale::Client', ->
   describe 'The authorize method', ->
     it 'should call the callback with a successful response', (done) ->
       client = new Client provider_key
-      client.authorize {app_key: application_key, app_id: application_id}, (response) ->
+      client.authorize {service_id: service_id, app_key: application_key, app_id: application_id}, (response) ->
```

Comment:
I think you should also add the `service_id` in the other calls below too.  There's another `authorize` and a `report` in this test file.